java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -641,7 +660,7 @@ public void shouldPublishMultipleStatusReportsToWebSocketForSubQueries() throws
             // Process Query
             processQuery(query);
             processQueriesFromQueue(1);
-            processQueriesFromQueue(1);
+            processLeafPartitionQuery(5);
```

Comment:
It's quite difficult to tell what is going on here. The call to processQueriesFromQueue(1) includes a call to processLeafPartitionQuery, then processLeafPartitionQuery is called directly as well.

It looks like all the other usages of processQueriesFromQueue(int) have been removed. Can we remove this one as well?